### DISCUSSION TITLE
error starting server

### DISCUSSION THREAD

**Discussion by raryskin** [l3atbc-datadog/pushkin#97] [General] [answered]
`pushkin start;` gives me the following error:

> Bind for 0.0.0.0:80 failed: port is already allocated

Is this a docker issue?

**Comment by raryskin** [comment]
Nevermind! Problem solved. Turns out the docker tutorial was running on port 80 :)

**Comment by conbainbridge** [comment, marked as answer]
Glad to hear you were able to solve this! If other people run into similar port issues, you can see what is running at a given port using this command (at least in MacOS terminal): `lsof -i tcp:80`

It's also possible to kill processes running in this port using (replacing <PID> with the PID that gets printed from the lsof command above):
`kill -9 <PID>`

**Reply by jkhartshorne** [reply]
@conbainbridge - is this worth adding to 'troubleshooting'?

**Reply by conbainbridge** [reply]
Will do!

**Reply by jkhartshorne** [reply]
Looks like someone already posted this as an issue! https://github.com/pushkin-consortium/pushkin/issues/87
